tools/metricsdocs/metricsdocs.go

```diff
@@ -4,13 +4,13 @@ import (
 	"bytes"
 	"fmt"
 	"log"
+	"sigs.k8s.io/controller-tools/pkg/markers"
```

Comment:
can we move this to a separate category?

Reply:
done